docs/develop/convert-javascript-to-typescript.md

```diff
@@ -51,11 +75,6 @@ You can use the Office Add-in template in Visual Studio to create an add-in that
 
 1. From the **Tools** tab, choose **NuGet Package Manager** and then select **Manage NuGet Packages for Solution...**.
```

Comment:
I think this step needs to be removed from here.

Reply:
removed.